microsoft-edge/extensions-chromium/publish/api/addons-publish-api-reference.md

```diff
@@ -0,0 +1,373 @@
+---
+description: The Add-ons Publish API Reference, for REST endpoints to automate publishing updates to add-ons that are submitted to the Microsoft Edge Add-ons website.
+title: Add-ons Publish API Reference
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 08/10/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+keywords: edge-chromium, extensions development, browser extensions, add-ons, partner center, developer, publish api, add-ons publish api
+---
+# Add-ons Publish API Reference
+
+This is the REST endpoint reference for the Publish Add-ons API for Microsoft Edge Add-ons.  This API automates publishing updates to add-ons that have been submitted to the Microsoft Edge Add-ons website.
+
+
+<!-- ====================================================================== -->
+## Get the list of products
+
+Gets a list of all products that belong to the account.  
+
+Overview: [Getting the list of products](using-addons-publish-api.md#getting-the-list-of-products)
```

Comment:
I think this back linking to the tutorial is un-necessary and will create a circular loop. Users will go to the Tutorial, click API reference and come to this, and again this link will take them back to the tutorial.

Rather, we should add a sentence at the beginning or the end of the API reference, saying Refer to "Using the APi" documentation for a tutorial on how to use these API's.